statuses.py

```diff
@@ -81,4 +81,7 @@
    "Keep calm and give a hug.",
    "Start your day with a hug! 🤗",
    "Feel the presence of love wrapped up within a hug.",
+   "Better than esperanto, hugs are a universal language.",
+   "How can we renounce the cuddly habits, the comfort, the indolent well-being of life? - François René de Chateaubriand / Mémoires d’outre-tombe ",
```

Comment:
In the quotes that are quoting someone, I like it better when the person being quoted is in their own line (see the quote on line 70 for example).

Also, there seems to have been something lost in translation? I'm not sure what the word "indolent" is meant to be here, are there maybe other suggested translations for the original French?

Reply:
It's an old form of the same french word "**indolent**" (_plural here_), It's more of a reference to an easy going life, without trouble and pain, not "**lazy**" or something similar that can be of a negative connotation of a bad habit.
I could perhaps use the term "**indulging**" instead in the context, if it "indolent" doesn't seem fitting in English.

NOTE: The work is from 1850 so the use of such expression may have been different.

Reply:
I see. It still came across as clumsy to me, so I searched and found an English translation of the work ([here](https://www.poetryintranslation.com/PITBR/Chateaubriand/ChateaubriandMemoirsBookVIII.php#anchor_Toc117934906)) and it seems like the quote goes something like "How to forego life’s tender usages, comforts, and indolent well-being?"

This seems to be less about cuddles and more about comfortable habits, so maybe let's leave this one out?

Reply:
Maybe it is, I focused on the "cuddly" habits after reading the response to comments in the original issue #12 regarding having the word "hug" in all quotes seeming like "compulsory". 
But sure, we can skip this one. I'll adjust the PR to it now.